generator/templates/engineProposal.template.js

```diff
@@ -19,7 +96,17 @@ import {Aave${protocolVersion}${chain}, Aave${protocolVersion}${chain}Assets} fr
  * - Snapshot: ${snapshot || "TODO"}
  * - Discussion: ${discussion || "TODO"}
  */
-contract ${contractName} is Aave${protocolVersion}Payload${chain} {
-
+contract ${contractName} is Aave${protocolVersion}Payload${chain} {`;
```

Comment:
```suggestion
contract ${contractName} is Aave${protocolVersion}Payload${chain} {\n`;
```

Maybe we can add this to prevent the next function on the same line? Though then we'd also need to add the proper spacing in the function renderers. 

Alternatively 
Not a big deal as the files will be formatted anyway.